### PR TITLE
zh: content_zh/docs/setup/kubernetes/prepare/requirements/index.md

### PR DESCRIPTION
**Pull request by zhangsean**
Translation for [#819](https://github.com/servicemesher/istio-official-translation/issues/819)

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @zhangsean, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/819) had been updated.

File: content_zh/docs/setup/kubernetes/prepare/requirements/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Pods 和服务
+description: 准备 Kubernetes Pods 和服务来运行启用了 Istio 的集群.
```
Comment (fleeto) [inline content_zh/docs/setup/kubernetes/prepare/requirements/index.md]: 预备 Kubernetes Pod 和服务，以便在启用了 Istio 的集群上运行。

File: content_zh/docs/setup/kubernetes/prepare/requirements/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - `https`
+    - `mongo`
+    - `redis`
+    - `tcp`
+    - `tls`
+    - `udp`
+
+    例如，`name: http2-foo` 或者 `name: http` 都是有效的端口命名，但 `name: http2foo` 就不是。如果端口名称不以特定前缀开头或者没有被命名，这个端口的流量将被作为简单 TCP 流量来处理，除非这个端口[明确地](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)使用 `Protocol: UDP` 来表示一个 UDP 端口。
+
+- **Pod 端口**：Pods 必须明确包含每个容器监听的端口列表。容器的每个端口都要使用 `containerPort` 配置。任何没有被列出的端口都将绕过 Istio 代理。
```
Comment (fleeto) [inline content_zh/docs/setup/kubernetes/prepare/requirements/index.md]: Pod 必须，中文里面的英文名词名词似乎不需要复数？另外注意大小写。

**Comment by SataQiu** [comment]
@zhangsean CI 没过，建议合并最新 master 分支代码，再推一次

**Pull request closed without merging**